.NET/Microsoft.Recognizers.Text.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27130.2026
+# Visual Studio Version 16
```

Comment:
Be careful to not make solution changes that are not essential to a PR. We should only change VS versions if really necessary.

Reply:
I have revised all the problems below according to your suggestions. Could you help me to review it again? Thanks for your help.